tests/test.py

```diff
@@ -1,4 +1,20 @@
 from pprint import pprint
-from trav import Travis
+from trav import Travis, get_svg_from_badge_url, get_travis_badge_url, TRAVIS_BASE_URL
 
 pprint(vars(Travis))
+
+
+def test_get_travis_badge_url():
+    result = get_travis_badge_url('coala', 'coala-bears', 'master')
+    assert result == TRAVIS_BASE_URL + 'coala/coala-bears.svg?branch=master'
+
+
+def test_get_svg_from_badge_url():
+    failing_build_svg = get_svg_from_badge_url(TRAVIS_BASE_URL + 'kx-chen/travis-fail.svg?branch=master')
+    assert 'failing' in failing_build_svg
```

Comment:
I am not sure if we want to be checking contents of SVG. It would be fine if we check for the response status code and ensure that the URL's are reachable.